python/truelayer_signing/__init__.py

```diff
@@ -25,20 +25,21 @@ def verify_with_pem(pkey: str) -> TlVerifier:
 
 
 def verify_with_jwks(
-    jwks: Mapping[str, List[Mapping[str, str]]], jws_header: Mapping[str, str]
+    jwks: Mapping[str, List[Mapping[str, str]]], jws_header: JwsHeader
```

Comment:
Should jwks be type hinted to `Dict` instead of `Mapping`? 

https://stackoverflow.com/questions/52487663/python-type-hints-typing-mapping-vs-typing-dict

Reply:
I chose `mapping` to keep the api as flexible as possible.